### PR TITLE
Collection view

### PR DESCRIPTION
**Pull request by ryantstone**
Wanted to do more but ran out of time 😢

### CODE REVIEW

File: BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Ryan/RyanViewController.swift [hunk 1]
```diff
@@ -10,10 +10,89 @@
...
+                self.collectionView.reloadData()
+            }
+        }
+    }
+//    lazy var layout: UICollectionViewLayout = {
+//        let layout = UICollectionViewLayout()
+//        layout
+//        layout.collectionViewContentSize = self.view.frame.width
+//        return layout
+//    }()
```
Comment (ivancr) [inline BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Ryan/RyanViewController.swift]: Dead code or will you be using it?

File: BackToBasicsUIKit/BackToBasicsUIKit.xcodeproj/xcuserdata/ryanstone.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist [hunk 2]
```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Bucket
```
Comment (ivancr) [inline BackToBasicsUIKit/BackToBasicsUIKit.xcodeproj/xcuserdata/ryanstone.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist]: Would you mind removing this breakpoints stuff from your commit? And while you’re at it, add it to gitIgnore?

**Review by ivancr — APPROVED** [review]

**Comment by ivancr** [comment]
Nice job @ryantstone. I love your coding style.

**Comment by ryantstone** [comment]
💥

**Pull request merged**